rest-api/7-secure-by-design/Domain/Exceptions/DomainPrimitiveArgumentException.cs

```diff
@@ -1,8 +1,5 @@
-using System.Runtime.Serialization;
-
 namespace Defence.In.Depth.Domain.Exceptions;
 
-[Serializable]
```

Comment:
@altenstedt Is there a reason to keep obsolete serialization of Domain exceptions? 
See https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0051